internal/handlers/handlers_test.go

```diff
@@ -130,3 +143,24 @@ func TestGetLotteryNumbers(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionHandler(t *testing.T) {
```

Comment:
It's great that you've added a test for the new `VersionHandler`. However, it would be even better if you could add a test case for when the `VersionHandler` fails to write the response. [medium]